proto/spacemesh/v1/smesher_types.proto

```diff
@@ -128,3 +128,17 @@ message PostSetupStatus {
     PostSetupOpts opts = 3; // setup options previously set by the user
     string error_message = 4; // The error message, if the state is STATE_ERROR
 }
+
+message SmeshingStatusRequest {
+    string smesher_id = 1;
```

Comment:
smesher id should be bytes

i think we are never building post or atxs for more than one smesher. why do you need smesher id in the request?

Reply:
yes, mistake, can use nodeID instead, if will need